.github/workflows/integration-package-release.yaml

```diff
@@ -0,0 +1,70 @@
+name: Publish integration package to PyPI
+
+on:
+  push:
+    tags:
+      - "prefect-\\w+-[0-9]+.*"
+
+jobs:
+  build-pypi-dists:
+    name: Build Python package
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.commit }}
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Extract package name
+        id: package_name
+        run: |
+          TAG_NAME=${{ github.ref_name }}
+          PACKAGE_NAME=$(echo "prefect-azure-0.3.8" | sed 's/-[0-9][.0-9]*$//')
```

Comment:
hmm flagging this while I think about this

Reply:
should this be
```console
PACKAGE_NAME=$(echo "$TAG_NAME" | sed 's/-[0-9][.0-9]*$//')
```

Reply:
Yup, should be fixed in https://github.com/PrefectHQ/prefect/pull/12806/commits/60cafe83fedc65f3ea64d0aadf20d30c2cc0ab4d